### DISCUSSION TITLE
Introducing the Novel SDK

### DISCUSSION THREAD

**Discussion by steven-tey** [steven-tey/novel#124] [Announcements]
Introducing the official `novel` SDK: https://novel.sh/sdk

![sdk](https://github.com/steven-tey/novel/assets/28986134/a7676f87-130f-4d88-b8c2-007a46d4d038)

You can now use Novel in your app with one single command:

```
npm i novel
```

Then, you can use it in your code like this:

```jsx
import { Editor } from "novel";

export default function App() {
  return <Editor />;
}
```

- Example app: https://github.com/steven-tey/novella
- Learn more: https://github.com/steven-tey/novel/blob/main/README.md#installation

Feedback & contributions welcome! 😄

**Comment by reveurguy** [comment]
Hello Steven!
Can we customize the SDK to manage things according to our needs?
Can we remove some features and customize some features according to our use using the SDK or would we have to implement it by scratch using the technologies Novel is built.

By the way, it's a great project. Thanks for building and sharing it!

**Reply by andrewdoro** [reply]
hey sry it took so long, there is a canary packages that should allow better customization.[0.2.0-canary.0](https://www.npmjs.com/package/novel/v/0.2.0-canary.0) It's basically exporting a series of building of blocks with some already configured extensions that can be overwritten. I still have a few things to cleanup but the idea is for almost full control on your side.
https://github.com/steven-tey/novel/discussions/273#discussioncomment-8441523
